pom.xml

```diff
@@ -23,7 +23,7 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
-  
+    
```

Comment:
???

Reply:
Probably a result of adding and removing dependencies that the formatter picked up on.